org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/utils/SonarLintUtils.java

```diff
@@ -36,6 +36,17 @@
 import org.sonarsource.sonarlint.core.commons.Language;
 
 public class SonarLintUtils {
+  /**
```

Comment:
I thought about enhancing the extension point to have it split for whitelisted standalone and connected mode languages but that doesn't work well if you have Eclipse without CDT for example: There should still be notification shown but due to CDT missing it wouldn't work out well.